jolt-core/src/jolt/vm/read_write_memory.rs

```diff
@@ -494,6 +498,7 @@ where
     fn read_leaves(&self, polynomials: &Self, gamma: &F, tau: &F) -> Vec<DensePolynomial<F>> {
         let num_ops = polynomials.a_read_write.len();
         let read_fingerprints = (0..num_ops)
+            .into_par_iter()
```

Comment:
Let's speed all 4 of these up with hashing improvements as well: https://github.com/a16z/Lasso/commit/b802e5da4cf6b881ed2840049aa39ade5fa863bf